grafana-plugin/src/containers/RemoteFilters/TimeRangePickerWrapper.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+
+import { dateMath, DateTime, TimeRange, toUtc } from '@grafana/data';
+import { TimeZone } from '@grafana/schema';
+import { TimeRangePicker } from '@grafana/ui';
+import { noop } from 'lodash';
+
+interface TimeRangePickerProps {
+  value: TimeRange;
+  timeZone: string;
+
+  onChange: (timeRange: TimeRange) => void;
+}
+
+export function TimeRangePickerWrapper(props: TimeRangePickerProps) {
+  const { timeZone, onChange } = props;
```

Comment:
This is a wrapper around `<TimeRangePicker/>` that is used extensively in grafana dashboards, but the component exposed by storybook doesn't have the built-in logic for `onMoveBackward`, `onMoveForward`, `zoom` etc. That component has this logic defined in `@grafana/scenes` instead, therefore some logic has been taken from there as they do not export these helpers